Project/feature-u-soundboard/src/features/tts/feature.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { createFeature } from 'feature-u';
+import TextToSpeechForm from './components/TextToSpeechForm';
+
+export default createFeature({
+  name: 'tts',
```

Comment:
Gebruik hier ook de featureName.ts conventie. We worden namelijk ook beoordeeld op code kwaliteit en dat is makelijke bonus punten.